meta/Search.php

```diff
@@ -376,6 +379,25 @@ public function findColumn($colname) {
         return $col;
     }
 
+    /**
+     * Check if a row is empty / only contains a reference to itself
+     *
+     * @param array $rowColumns an array as returned from the database
+     * @return bool
+     */
+    private function isRowEmpty($rowColumns) {
+        $C = 0;
+        foreach($this->columns as $col) {
+            $val = $rowColumns["C$C"];
+            $C += 1;
+            if (empty($val) || is_a($col->getType(),'dokuwiki\plugin\struct\types\Page') && $val == $rowColumns["PID"]) {
```

Comment:
A row containing a 0 might not be empty. I guess we should compare against ```=== ''```  here?